jira/client.py

```diff
@@ -1560,8 +1560,10 @@ def createmeta(
         return self._get_json("issue/createmeta", params)
 
     def _get_user_accountid(self, user):
-        """Internal method for translating an user to an accountId."""
+        """Internal method for translating an user to an accountId. Return None and -1 unchanged."""
         try:
+            if user is None or user in (-1, "-1"):
```

Comment:
I think the approach is good, my only suggestion might be to:

```suggestion
            if user in (None, -1, "-1"):
```

but not major

Reply:
Hey thanks for the feedback! I will try to address it shortly and resolve the conflicts.